python/actorkeys/enu.py

```diff
@@ -1,4 +1,4 @@
-KeysDictionary('enu', (4, 1),
+KeysDictionary('enu', (4, 2),
```

Comment:
What does this change mean?

Reply:
That's just the version number, the archiver tracks these.